esp-hal/src/gpio/placeholder.rs

```diff
@@ -6,9 +6,8 @@
 // polluting the main module.
 
 use super::*;
-use crate::gpio::interconnect::connect_input_signal;
 
-impl crate::peripheral::Peripheral for Level {
+unsafe impl crate::peripheral::Peripheral for Level {
```

Comment:
Not sure if this is an "interconnect signal" (as per the PR title), so I'll just ask if you leaving this `impl` here was intentional.

Reply:
Nah, this was a mistake, thanks for spotting